meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-kernel/linux/linux-lmp-ti-staging_git.bb

```diff
@@ -6,8 +6,8 @@ KERNEL_REPO ?= "git://git.ti.com/git/ti-linux-kernel/ti-linux-kernel.git"
 KERNEL_REPO_PROTOCOL ?= "https"
 KERNEL_BRANCH ?= "ti-linux-6.1.y"
 
-LINUX_VERSION ?= "6.1.69"
-SRCREV_machine ?= "2233af66faf7b81b6c286285e50cda5595dc410d"
```

Comment:
Please describe in your git commit why the change, is it aligning with a certain meta-ti commit? A certain BSP release?

Reply:
I tried my best